asciimatics/screen.py

```diff
@@ -1914,6 +1914,7 @@ def close(self, restore=True):
                 curses.echo()
                 curses.nocbreak()
                 curses.endwin()
+                signal.signal(signal.SIGWINCH, signal.SIG_DFL)
```

Comment:
Isn't it actually safer to reinstate the previous signal handler than to assume that the default was the one previously active?